code/pages/Site.php

```diff
@@ -156,6 +156,9 @@ public function onBeforeWrite() {
 			}
 		}
 
+		//Set MenuTitle to NULL so that Title is used
+		$this->MenuTitle = NULL;
```

Comment:
Is it something that users can change via the CMS ? Or is it that it gets set once on first write, but changing the site Title later doesn't update MenuTitle ?